aire-ui/src/main/java/io/sunshower/zephyr/ui/rmi/Parameters.java

```diff
@@ -0,0 +1,19 @@
+package io.sunshower.zephyr.ui.rmi;
+
+import lombok.val;
+
+public class Parameters {
+
+  @SuppressWarnings("unchecked")
+  public static String constructExpression(String methodName, Class<?>... values) {
+
+    val result = new StringBuilder("this.").append(methodName).append("(");
```

Comment:
I'm not sure if I hate this or I love it

Reply:
eh--works